### DISCUSSION TITLE
Shut off BECCS techniques

### DISCUSSION THREAD

**Discussion by daleihao** [JGCRI/gcam-core#440] [Q&A] [answered]
Hi to all, I want to shut off BECCS techniques for all sectors. Does anyone know how to imelement it in GCAM?

Kind regards,

**Comment by ouyang363** [comment]
sector by sector modifyng shwt (easier to do this in the data system)

**Comment by pkyle** [comment]
Here is a screengrab identifying the technologies whose share-weights should be set to zero, either in gcamdata (changing A22.globaltech_shrwt.csv and other data tables and re-building xml's), or in an extra add-on xml file. This includes some technologies that are in GCAM-USA but not the standard configuration.
<img width="960" alt="Screenshot 2024-06-18 at 11 06 05 PM" src="https://github.com/JGCRI/gcam-core/assets/20807787/2a75b680-6d08-4ee1-83d8-eea0719c53a4">

**Reply by daleihao** [reply]
Thank you, Page! I have some following questions:


1. In the `A22.globaltech_shrwt.csv` as below, how can I identify the technologies belonging to BECCS? The technology name with CCS?
<img width="1263" alt="Screenshot 2024-06-19 at 12 00 02 PM" src="https://github.com/JGCRI/gcam-core/assets/52718138/c70363f8-4cde-4ab5-8bf8-045f7aee655e">

2. How can I find other data tables to modify the shareweights for BECCS?

3. I found that there is `high_cost_ccs.xml` in the `gcamdata/xml`, which set  `<input-cost>10000</input-cost>`. Can I use it to shut off css as a simply way?

4. I also found that comparing `ccs_supply_high.xml` and `ccs_supply_ccs_supply_lowest.xml`, their `extractioncost` are different. Can I set `extractioncost` as a high value to shut off ccs?

**Comment by pkyle** [comment, marked as answer]
1. The BECCS technologies are listed in that screengrab I sent
2. The relevant shareweight tables are all in the energy folder and have the string `shrwt` in their filenames.
3. That file will effectively shut off all ccs, not just beccs. If that is what is wanted in your study design, then you can use that file.
4. Sure. The cost of CCS for any technology consists of the sum of the technology cost, the carbon storage cost (a flat marginal cost assigned to the carbon-storage sector), and the resource cost. The cost parameter for the resource is called `extractioncost`. This cost increases with deployment as CO2 storage reservoirs are filled up. But you could shut off or reduce deployment of CCS on a cost basis by increasing the assumed costs at any of these three places.

**Reply by daleihao** [reply]
Got it. Thank you very much!

**Reply by choiHenry** [reply]
This answer helped me a lot especially understanding the cost components of CCS technologies.
